python/tools/recorded-call-graph-metrics/README.md

```diff
@@ -0,0 +1,17 @@
+# Recorded Call Graph Metrics
+
+also known as _call graph tracing_.
+
+Execute a python program and for each call being made, record the call and callable. This allows us to compare call graph resolution from static analysis with actual data -- that is, can we statically determine the target of each actual call correctly.
+
+This is still in the early stages, and currently only support a very minimal working example (to show that this approach might work).
```

Comment:
```suggestion
This is still in the early stages, and currently only supports a very minimal working example (to show that this approach might work).
```
I might be wrong here given that English is not my first language.
Or should it be "and we currently only ..."?

Reply:
no that sounds about right to me as well :+1: